mypy/build.py

```diff
@@ -430,10 +430,11 @@ def __init__(self, data_dir: str,
                  reports: Reports,
                  options: Options,
                  version_id: str,
-                 plugin: Plugin) -> None:
+                 plugin: Plugin,
+                 errors: Errors) -> None:
```

Comment:
It was mentioned already, but moved to another place where it might be easier to find.